src/query/src/error.rs

```diff
@@ -110,6 +112,19 @@ pub enum Error {
         table_name: String,
         location: Location,
     },
+
+    #[snafu(display("Failed to convert value to sql value: {}", value))]
+    ConvertSqlValue {
+        value: Value,
+        #[snafu(backtrace)]
+        source: sql::error::Error,
+    },
```

Comment:
nitpick: add a new line?
```suggestion
    },

```